addOns/paramminer/src/main/java/org/zaproxy/addon/paramminer/wordlists/smalllist.txt

```diff
@@ -0,0 +1,835 @@
+AID
```

Comment:
Should be added under `src/main/resources` instead of `src/main/java`.

Reply:
If this is meant to be deployed to ZAP's home dir it should go to `src/main/zapHomeFiles` instead.

Reply:
Done

Reply:
> If this is meant to be deployed to ZAP's home dir it should go to src/main/zapHomeFiles instead.

Uhh not exactly. There will be many other wordlist joining in too :) and it's almost specific to this extension